DegenPrimer/AllSecStructures.py

```diff
@@ -52,8 +52,8 @@ def __init__(self, abort_event, job_id, primers):
         self._equilibrium_concentrations = None
         #reports
         self._job_id = job_id
-        self._short_structs_filename = job_id+'-structures-short.txt'
-        self._full_structs_filename  = job_id+'-structures-full.txt'
+        self._short_structs_filename = job_id[-20:] + '-structures-short.txt'
+        self._full_structs_filename  = job_id[-20:] + '-structures-full.txt'
```

Comment:
This is not a proper way to do it since it truncates the hash appended to the `job_id` to distinguish runs with different parameters. If the job_id is too long for a file system, it's best to introduce the `--job-id` option that would overwrite `job_id` generation from primer ids here: https://github.com/allista/DegenPrimer/blob/448c2035c965f694b54dc39be203d4f0a8867ee3/DegenPrimer/DegenPrimerConfig.py#L650